src/coreclr/tools/dotnet-pgo/PgoRootCommand.cs

```diff
@@ -0,0 +1,296 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Help;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Diagnostics.Tools.Pgo
+{
+    internal sealed class PgoRootCommand : RootCommand
+    {
+        public Option<string[]> InputFilesToMerge { get; } =
+            new(new[] { "--input", "-i" }, "Input .mibc files to be merged. Multiple input arguments are specified as --input file1.mibc --input file2.mibc") { IsRequired = true, Arity = ArgumentArity.OneOrMore };
+        public Option<string[]> InputFilesToCompare { get; } =
+            new(new[] { "--input", "-i" }, "The input .mibc files to be compared. Specify as --input file1.mibc --input file2.mibc") { IsRequired = true, Arity = new ArgumentArity(2, 2) /* exactly two */ };
+        public Option<string> InputFileToDump { get; } =
+            new(new[] { "--input", "-i" }, "Name of the input mibc file to dump") { IsRequired = true, Arity = ArgumentArity.ExactlyOne };
```

Comment:
I wanted to use `Argument<T>` instead of `Option<T>`, but it turned out `Argument<T>` does not support aliases (`-i:path/to/file` which we are using).